server/portal/apps/workspace/api/views.py

```diff
@@ -324,7 +353,9 @@ def getPrivateApps(self, user):
         return my_apps
 
     def getPublicApps(self, user):
-        # TODOv3: make tapipy request for public apps to compare against apps in AppTrayEntry
+        # TODOv3: Ensure `listType` only includes PUBLIC apps
+        tapis = user.tapis_oauth.client
+        apps_listing = tapis.apps.getApps(select="version,id,notes", search="(enabled.eq.true)", listType="SHARED_PUBLIC")
```

Comment:
I'm getting an empty list here so I'm missing my apps i added to my model ( i have a 0.1 hellow world and then a hello-world without a version)

Reply:
Hmm, this should be a public only listing unrelated to the model. This query produces no results? And in the portals.develop tenant?

Reply:
Ah I found it issue. The `0.1` hello world didn't work, since the version is `0.0.1`, and the non-versioned hello world wasn't loading because of the other issue you mentioned regarding not including version in the filtering process. Should be fixed now

Reply:
yeah, i'm getting an empty list on:
>  apps_listing = tapis.apps.getApps(select="version,id,notes", search="(enabled.eq.true)", listType="SHARED_PUBLIC")

Reply:
(resolved, required a fresh `make build`)